packages/react-components/react-table/src/components/Table/Table.types.ts

```diff
@@ -26,19 +21,7 @@ export type TableContextValues = {
 /**
  * Table Props
  */
-export type TableProps = ComponentProps<TableSlots> & {} & Partial<TableContextValue> & {
-    /**
-     * Called when the sorted column changes
-     */
-    onSortColumnChange?: (
-      e: React.MouseEvent<HTMLElement> | React.KeyboardEvent<HTMLElement>,
-      data: { sortState: SortState },
-    ) => void;
-
-    sortState?: SortState;
-
-    defaultSortState?: SortState;
-  };
+export type TableProps = ComponentProps<TableSlots> & {} & Partial<TableContextValue>;
```

Comment:
```suggestion
export type TableProps = ComponentProps<TableSlots> & Partial<TableContextValue>;
```

Do we need this `{}`? 😅

Reply:
nope